mod/wikirate_companies/set/type_plus_right/region/oc_jurisdiction_key.rb

```diff
@@ -5,3 +5,11 @@ def oc_code
 event :clear_jurisdiction_key_cache do
   Card::Region.cache.reset_all
 end
+
+event :validate_jurisdiction_key, :validate,
+      on: :save, changed: :content do
+  oc_key = Card.search left: { type: :region },
+                       right: :oc_jurisdiction_key,
+                       content: card.content
```

Comment:
Technically this search will return an array (a list of cards). So even though it doesn't matter, it's slightly clearer to say `oc_keys`.